doc/zsh-completion.zsh

```diff
@@ -374,7 +374,7 @@ function _restic_forget {
     '*--host[only consider snapshots with the given `host` (can be specified multiple times)]:' \
     '--tag[only consider snapshots which include this `taglist` in the format `tag[,tag,...]` (can be specified multiple times)]:' \
     '*--path[only consider snapshots which include this (absolute) `path` (can be specified multiple times)]:' \
-    '(-c --compact)'{-c,--compact}'[use compact format]' \
+    '(-c --compact)'{-c,--compact}'[compact printed output]' \
```

Comment:
Please remove this change, as the completion files are generated automatically by the release build scripts.